platform-tests/src/acceptance/rds_broker_test.go

```diff
@@ -134,3 +243,48 @@ func quietCf(program string, args ...string) *Session {
 	Expect(err).NotTo(HaveOccurred())
 	return command
 }
+
+type RDSClient struct {
+	region string
+	rdssvc *rds.RDS
+}
+
+func NewRDSClient(region string) (*RDSClient, error) {
+	sess, err := session.NewSession(&aws.Config{Region: aws.String(region)})
+	if err != nil {
+		fmt.Println("Failed to create AWS session,", err)
+		return nil, err
+	}
+
+	rdssvc := rds.New(sess)
+	return &RDSClient{
+		region: region,
+		rdssvc: rdssvc,
+	}, nil
+}
+
+func (r *RDSClient) GetDBFinalSnapshots(ID string) (*rds.DescribeDBSnapshotsOutput, error) {
+	params := &rds.DescribeDBSnapshotsInput{
+		DBSnapshotIdentifier: aws.String(ID + "-final-snapshot"),
```

Comment:
This identifier should really be generated in one place rather than having logic duplicated on https://github.com/alphagov/paas-cf/pull/508/files#diff-ebbb4adbeb7f9ea59f6a7015a3a3afbaR281 to avoid future errors if we change it. Again, not enough to block merging this.